pkg/noun/trace.c

```diff
@@ -1218,6 +1223,7 @@ u3t_sstack_push(u3_noun nam)
 void
 u3t_sstack_pop()
 {
+	if (  !stk_u ) return;
```

Comment:
indentation is wrong here (maybe tabs instead of spaces?)

Reply:
removed